### PR TITLE
changes for 0.9.0

### PR DESCRIPTION
**Pull request by Noviny**

### CODE REVIEW

File: documentation/assets/banner.jpg
Comment (dubeyShivank) [inline documentation/assets/banner.jpg:1]: Can we omit these three files under `documentation/assets/*.jpg`?

File: package.json [hunk 1]
```diff
@@ -19,7 +22,7 @@
     "ui library"
   ],
   "peerDependencies": {
-    "@voxeet/voxeet-web-sdk": "3.8.0",
+    "@voxeet/voxeet-web-sdk": "^3.9.0-beta.3",
```
Comment (dubeyShivank) [inline package.json:25]: Move to stable voxeet web sdk version?

File: src/components/conference/JoinConferenceButton/JoinConferenceButton.tsx [hunk 2]
```diff
@@ -27,26 +33,42 @@ const JoinConferenceButton = ({
...
-    }
+    try {
+      if (onInitialise) {
+        onInitialise();
+      }
+      const newConference = await createConference({
+        alias: meetingName,
+        params: {
+          // Keep empty conferences alive for 5 minutes to allow participants to join or rejoin
+          ttl: 60 * 5,
```
Comment (dubeyShivank) [inline src/components/conference/JoinConferenceButton/JoinConferenceButton.tsx:52]: @vincentsong @Noviny enhancement idea for a subsequent release - can `ttl` be an optional input param?

Comment (Noviny) [reply src/components/conference/JoinConferenceButton/JoinConferenceButton.tsx:52]: Definitely.

**Pull request closed without merging**